package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Themeroller",
+  "version": "0.0.0",
+  "description": "This project is a themeroller for css-chassis",
+  "main": "index.js",
+  "scripts": {
+    "test": "grunt"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jquery/themeroller.css-chassis.com"
+  },
+  "keywords": [
+    "Themeroller",
+    "css-chassis"
+  ],
+  "author": "jQuery foundation and other contributors",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/jquery/themeroller.css-chassis.com/issues"
+  },
+  "dependencies": {
+    "express": "~4.13.0"
```

Comment:
In most of our projects we hardcode dependency versions. We should probably discuss if we want to do that here, and if not, let's use a caret :)